package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@libero/eslint-config",
+  "version": "0.0.1",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:libero/eslint-config.git"
+  },
+  "author": "libero-npm@elifesciences.org",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "build": "tsc",
+    "prettier": "prettier --write 'src/**/*ts'"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.17.0",
+    "@typescript-eslint/parser": "^2.17.0",
+    "eslint": "^6.8.0",
```

Comment:
Since this package brings in `eslint`, it's not just `config`

Reply:
Or does it? It's also in the `peerDependencies `which suggests the project using this will require `eslint` on his own